docs/Operators-ml.md

```diff
@@ -690,46 +698,44 @@ This version of the operator has been available since version 1 of the 'ai.onnx.
 
 <dl>
 <dt><tt>offset</tt> : list of floats</dt>
-<dd>first, offset by this, must be same length as scale</dd>
+<dd>First, offset by this. Can be length of features or length 1, in which case it applies to all features</dd>
```

Comment:
Does it mean that the input shape must be [N, C] (where C is the feature length)? If I have an input shape [N, C, H, W], this statement looks not very clear.

Reply:
Right. Clarified to indicate special case for [N,F] situation.

Reply:
Do you mean that Scaler only accepts [N,C]-inputs? If so, we need to say something like "This operator only consumes 2-D inputs" in its document string. If it can handle both of 1-D (aka [C]) and 2-D (aka [N, C]) inputs, we can add "This operator only consumes 1-D and 2-D inputs. The right-most axis is considered as the feature dimension."